ChangeLog.d/8060.txt

```diff
@@ -0,0 +1,4 @@
+Features
+    * CIPHER_C dependency of MBEDTLS_CCM_C and MBEDTLS_GCM_C has been removed
+      in favor of BLOCK_CIPHER_C. The latter is also automatically enabled
```

Comment:
Minor: I'd rather not mention the internal module in the ChangeLog. Generally speaking, ChangeLog entries should adopt the user's point of view, and explain what's now possible that wasn't before, rather than give details about how we went to implement it.

Suggestion: "The CCM and GCM modules no longer depend on MBEDTLS_CIPHER_C. People who use CCM and GCM but don't need the Cipher API can now disable MBEDTLS_CIPHER_C in order to save code size."